sdk/anomalydetector/ai-anomaly-detector-rest/src/models.ts

```diff
@@ -3,9 +3,19 @@
 
 /** The request of entire or last anomaly detection. */
 export interface DetectRequest {
-  /** Time series data points. Points should be sorted by timestamp in ascending order to match the anomaly detection result. If the data is not sorted correctly or there is duplicated timestamp, the API will not work. In such case, an error message will be returned. */
+  /**
+   * Time series data points. Points should be sorted by timestamp in ascending
+   * order to match the anomaly detection result. If the data is not sorted
+   * correctly or there is duplicated timestamp, the API will not work. In such
+   * case, an error message will be returned.
+   */
   series: Array<TimeSeriesPoint>;
-  /** Optional argument, can be one of yearly, monthly, weekly, daily, hourly, minutely, secondly, microsecond or none. If granularity is not present, it will be none by default. If granularity is none, the timestamp property in time series point can be absent. */
```

Comment:
It seems that the document format is slightly different.

Reply:
may because I run rushx lint && rushx format for them.